### PR TITLE
fix tests so one can execute them from a clean repo

### PR DESCRIPTION
**Pull request by 5c077yP**
Hey, 

i couldn't get the tests running locally when i just cloned the repo, installed dependencies (`pip install -r requirements.txt`) and run tests `./quick_test.sh` and `test.sh`. 

i wanted to contribute a PR for #1534 but i need this to get the tests running from the beginning

### CODE REVIEW

File: test/etc/test_sslv3_disabled/shinken.cfg [hunk 1]
```diff
@@ -114,7 +114,7 @@ local_log=/dev/null
 # By default don't launch even handlers during downtime. Put 0 to
 # get back the default N4G105 behavior
 no_event_handlers_during_downtimes=1
-
+modules_dir=modules
```
Comment (Seb-Solon) [inline test/etc/test_sslv3_disabled/shinken.cfg]: already line 41 actually

**Comment by Seb-Solon** [comment]
Actually the test_sslv3 is disabled. the ssl.SSLContext is not available in python<=2.7.9.

Somehing else, I dont get why you change de import order. 

The RunTimeError fix is legit to me, I don't get why it was like this before

**Comment by 5c077yP** [comment]
concerning the import order. for me it was important to have the import shinken_test first as this is importing shinken properly (call to __import_shinken). otherwise the tests were failing and it told me that e.g. shinken has no module such as brok. 

i just cloned the repo , did the pip install -r requirements.txt and called to `quick_test.sh`

**Comment by gst** [comment]
I'm also ok with this PR..

**Comment by naparuba** [comment]
yes 2


thanks.

**Pull request merged**